docs/pages/machine-id/deployment/github-actions.mdx

```diff
@@ -80,6 +80,22 @@ spec:
 ```
 </Admonition>
 
+<Admonition type="note" title="Using GitHub Enterprise Cloud?">
```

Comment:
Two Admonitions in a row might get busy. I'd make sure this looks okay on the dev server.

Reply:
What options do we have if this looks too busy ? Tabs ?